### ISSUE TITLE
trying edition with radedit

### ISSUE THREAD

**Issue by MichelPezzat** [nktoan/h-edit#3]
Thank you for sharing your work, Toan. Results are amazing.

I've been trying 2 use your work to edit chest x-ray images by slightly modyifing your scripts with the radedit pipeline. But I only get black images.
https://drive.google.com/drive/folders/1R7fkIb-hxlQW3vXHZNSq2Fk_QNM-l07P?usp=sharing
What am I missing? 
Thank you in advance.

Regards
Michel

**Comment by phamtrongthang123** [comment]
Try this script.
[main_demo_radedit.py](https://github.com/user-attachments/files/25615118/main_demo_radedit.py)

1. I stopped running VAE encode/decode under mixed precision (autocast) and used normal torch.no_grad() instead. SDXL VAE in fp16 is a common source of black outputs.
   See main_demo_radedit.py:251 and main_demo_radedit.py:344.
2. I aligned scheduler behavior with RadEdit (timestep_spacing="trailing" + set_alpha_to_one=False) and stopped forcing "leading" in the loop. That avoids latent trajectory mismatch that can collapse outputs.
   See main_demo_radedit.py:72 and main_demo_radedit.py:235.

Additional stabilizers I also did:
- I decoded/postprocessed via the pipeline image processor instead of the manual image_grid path (main_demo_radedit.py:347).
- I used deterministic latent encode (mode() vs sample()), mainly for faithfulness/reproducibility (main_demo_radedit.py:253).


Run:
```bash
python main_demo_radedit.py \
    --device_num 0 \
    --data_path hedit/support_devices \
    --output_path ./results/radedit_demo \
    --mode h_edit_R \
    --implicit \
    --num_diffusion_steps 50 \
    --skip 12 \
    --eta 1.0 \
    --cfg_src 1.0 \
    --cfg_src_edit 2.0 \
    --cfg_tar 3.0 \
    --optimization_steps 1 \
    --weight_reconstruction 0.1 \
    --latent_blend 0.75
```

My demo.yaml is just a bunch of 
```yaml
- image: /baseline/baseline/orig_seed000000.png
  source_prompt: "a chest x-ray, clinical radiology style, showing support devices"
  target_prompt: "a chest x-ray, clinical radiology style, no acute finding"
  editing_instruction: "remove support devices"
  blended_word: ""
```
